hermes-api/src/main/java/pl/allegro/tech/hermes/api/SubscriptionPolicy.java

```diff
@@ -77,14 +91,17 @@ public static SubscriptionPolicy create(Map<String, Object> properties) {
                 (Boolean) properties.getOrDefault("retryClientErrors", false),
                 (Integer) properties.getOrDefault("messageBackoff", DEFAULT_MESSAGE_BACKOFF),
                 (Integer) properties.getOrDefault("inflightSize", DEFAULT_INFLIGHT_SIZE),
-                (Integer) properties.getOrDefault("sendingDelay", DEFAULT_SENDING_DELAY)
+                (Integer) properties.getOrDefault("sendingDelay", DEFAULT_SENDING_DELAY),
+                ((Number) properties.getOrDefault("backoffMultiplier", DEFAULT_BACKOFF_MULTIPLIER)).doubleValue(),
```

Comment:
why not a Duble ?

Reply:
Hermes console may send both double and integer because it cuts trailing zeros (e.g. 3.0 => 3). The only way to prevent this is by sending string which I decided to avoid. JsonCreator treats 3 as Integer and an exception is thrown when casting to double.